### DISCUSSION TITLE
footer的两个shields图标，如何并排？

### DISCUSSION THREAD

**Discussion by tkfanswind** [spencerwooo/onedrive-vercel-index#413] [Q&A] [answered]
footer使用的两个shields图标，如何并排？ 现在是分两排显示，如何显示在一排

代码如下
<div align="center"><a href="https://xxx.cf"><img src="https://img.shields.io/badge/xxx%20One%20Drive-000000?logo=Apple&logoColor=FFFFFF" alt="xxx"></a><a href="https://xxx2.cf"><img src="https://img.shields.io/badge/xxx2%20One%20Drive-000000?logo=Apple&logoColor=FFFFFF" alt="xxx2"></a></div>

**Comment by zkeq** [comment, marked as answer]
在 `footer` 挂载 `shields` 的 `img` 标签中添加 `style` 属性

```
style=\"display: inline-block\" 
```

完整代码示例：

```
<img style=\"display: inline-block\" src=\"https://img.shields.io/badge/Vercel-black?style=flat&logo=Vercel&logoColor=white\" alt=\"vercel\" />
```

使用以上代码的示例站点：https://cdn.dream-plan.cn

如若更改合适行距  可在 `\styles\globals.css` 最后追加

```
.p-4.text-xs.font-medium.text-gray-400.w-full.text-center.border-t.border-gray-900\/10.dark\:border-gray-500\/30 {
  line-height: 1.3rem;
}
```

合适行距示例站点：

https://tuostudy.vercel.app
